srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java

```diff
@@ -187,7 +213,8 @@ private void discountBooksWithMoreThan111Stock(Books b, boolean premium) {
 
 	private void loadStockIfNotSet(Books b) {
 		if (b.getId() != null && b.getStock() == null) {
-			b.setStock(db.run(Select.from(BOOKS).byId(b.getId()).columns(Books_::stock)).single(Books.class).getStock());
+			b.setStock(
+					db.run(Select.from(BOOKS).byId(b.getId()).columns(Books_::stock)).single(Books.class).getStock());
```

Comment:
We could just add "stock" also [here](https://github.com/SAP-samples/cloud-cap-samples-java/pull/439/commits/47fbbbdd3a2cb670adaf78b288bee87d6ef613d2#diff-a1d8a452c4e97ab688ec6b9ad6cd40c2c5129a4f73ba2a1b0356e98904555a13R86)